piff/config.py

```diff
@@ -296,6 +318,10 @@ def meanify(config, logger=None):
 
     coords0 = np.array([u0.reshape(-1), v0.reshape(-1)]).T
 
+    # remove any entries with counts == 0
```

Comment:
I think this comment is not strictly true, in that there are additional other reasons for a box to be non-finite :)

Reply:
Yes indeed, I specified in the last commit that it was removing bin with non finite value inside